fabric-common-tags-api-v1/src/main/java/net/fabricmc/fabric/api/tags/v1/CommonFluidTags.java

```diff
@@ -0,0 +1,21 @@
+package net.fabricmc.fabric.api.tags.v1;
+
+import net.minecraft.fluid.Fluid;
+import net.minecraft.tag.TagKey;
+
+import net.fabricmc.fabric.impl.v1.TagRegistration;
+
+public class CommonFluidTags {
+	public static final TagKey<Fluid> LAVA = register("lava");
+	public static final TagKey<Fluid> WATER = register("water");
+	public static final TagKey<Fluid> MILK = register("milk");
+	public static final TagKey<Fluid> HONEY = register("honey");
```

Comment:
How would these tags include fluids that are not available as actual fluids in vanilla Minecraft?

Reply:
Vanilla has concepts for milk and honey, and are reasonably common for mods to make them be real fluids.

Reply:
What dex said. See: bumblezone with its honey fluid. Good thing I can see the future as I already got my fluid in c:honey
https://github.com/TelepathicGrunt/Bumblezone-Fabric/blob/latest-released/src/main/resources/data/c/tags/fluids/honey.json

Reply:
Would this API actually help in implementing the behavior dependent on tagged fluids? At most, these tags should be used in data generation rather than in `isIn` checks.

To me, specific functionality tags make far more sense than broad, meaningless 'common' tags. As such, functionality tags should be prioritized by any tag APIs.

Reply:
Implementing functionality can be for another module or actual mods, that is not the goal of this module.

This module is primarily aiming to identify or create tags that mods have adopted, either for recipes, functionality, or world generation, to make them more discoverable to promote mod compatibility. Tags are useless if you have to guess the name and/or purpose of it and get it wrong, or don't even know it exists.

Datagen including vanilla content into these tags is part of this module's goal.

Reply:
Tags should define functionality, not the other way around. I am opposed to any data generation automatically populating these tags.

Reply:
Feel free to make that module then! It would definitely be useful, but is out of scope for this one.

Reply:
This module shouldn't exist.